shell/platform/linux/fl_binary_messenger_test.cc

```diff
@@ -384,3 +385,96 @@ TEST(FlBinaryMessengerTest, ReceiveMessage) {
   // Blocks here until response_cb is called.
   g_main_loop_run(loop);
 }
+
+struct RespondsOnBackgroundThreadInfo {
+  FlBinaryMessenger* messenger;
+  FlBinaryMessengerResponseHandle* response_handle;
+  GMainLoop* loop;
+};
+
+static gboolean cleanup_responds_on_background_thread_info(gpointer user_data) {
+  RespondsOnBackgroundThreadInfo* info =
+      static_cast<RespondsOnBackgroundThreadInfo*>(user_data);
+  GMainLoop* loop = info->loop;
+
+  g_object_unref(info->messenger);
```

Comment:
It's generally safer to use `g_clear_object` here - that was any accidental access of this object will hit NULL install of invalid pointers.

Reply:
done